modules/portal/Portal.tsx

```diff
@@ -71,11 +73,34 @@ const Portal: React.FC<IProps> = ({
   // If window size changes, recalculate position based on new parentRef position
   // This effect also sets the initial position
   useEffect(() => {
-    target.setAttribute(
-      "style",
-      getPortalStyles(parentRef, direction, useParentWidth)
-    );
-  }, [direction, parentRef, target, useParentWidth, windowSize]);
+    const updateTargetStyle = () => {
+      target.setAttribute(
+        "style",
+        getPortalStyles(parentRef, direction, useParentWidth)
+      );
+    };
+    updateTargetStyle();
+    if (scrollableRootElement) {
+      const rootElement =
+        typeof scrollableRootElement === "string"
+          ? document.getElementById(scrollableRootElement)
```

Comment:
Isn't it possible to use a React Ref instead of accessing the DOM directly?

Reply:
the root element is on the actually html file on the project's root :/ 
> https://github.com/CarlsbergGBS/dmbuddy-offshift/blob/master/public/index.html#L82

Reply:
Oh I see. So this means that in other site, if the upper scroll element is not the root, we would create an extension of this component that changes the default scrollableRootElement right?

Reply:
we won't need an extenstion since we could just pass a prop `scrollableRootElement` with the element we want . I just make 'root' the default since most of the cases in our projects `root` is the element we want